test/remote/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM golang:1.16.2 AS go
```

Comment:
Could you please add a comment mentioning why this is needed, and where this is consumed ?

Reply:
Read further, I understand why :)